cddl/raw-integral-type-choice.cddl

```diff
@@ -0,0 +1,5 @@
+raw-integral-type-choice /= integral / tagged-integral-range
+integral-range = [min: opt-integral, max: opt-integral]
+tagged-integral-range = #6.564(integral-range)
+integral = int / #6.2(bytes) / #6.3(bytes)
```

Comment:
maybe, to make the CDDL more transparent to the readers:
```suggestion
biguint = #6.2(bytes)
bignint = #6.3(bytes)
bigint = biguint / bignint
integer = int / bigint
```
or (more condensed):
```suggestion
bigint = #6.2(bytes) / #6.3(bytes)
integer = int / bigint
```